eZ/Publish/Core/Repository/TrashService.php

```diff
@@ -107,13 +108,18 @@ public function loadTrashItem($trashItemId)
      * Content is left untouched.
      *
      * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException if the user is not allowed to trash the given location
+     * @throws \eZ\Publish\API\Repository\Exceptions\ForbiddenException if the user is trying trash the root location
```

Comment:
doc change will be needed on API and SignalSlot as well. And rest should be moved to the real integration tests in `/API/`, not the deprecated Core integration tests.